src/lolex-src.js

```diff
@@ -394,6 +398,12 @@ function hijackMethod(target, method, clock) {
     target[method].clock = clock;
 }
 
+exports.shouldAdvanceTime = false;
```

Comment:
not super in-love with this, but on the other hand I also hate magic constructor argument matching. What's wrong with config objects?